### PR TITLE
Added heart disease dataset and binary classification example.

### PR DESCRIPTION
**Pull request by KianYang-Lee**
When you added a new program / programs , put a cross on the  OS that you have test it on  
(Example: [ X ] Windows  )

[ X] Windows  
[  ] Linux Ubuntu  
[  ] Mac  
[  ] Others  (State here -> xxx )  

Assign a reviewer to review your code.

### CODE REVIEW

File: dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java [hunk 1]
```diff
@@ -0,0 +1,212 @@
+package ai.certifai.training.classification;
```
Comment (codenamewei) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java:1]: license header before package

File: dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+import org.nd4j.linalg.learning.config.Nesterovs;
+import org.nd4j.linalg.lossfunctions.LossFunctions;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class heartDiseaseBinaryClassification {
```
Comment (codenamewei) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java:41]: Capital for each word

File: dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    private static final int totalData = 303;
+    private static final double ratioTrainTestSplit = 0.8;
+
+    // Training info
+    private static final int epoch = 1000;
```
Comment (codenamewei) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java:46]: normal convention for static final variables is capital letter for all
if there's more than one word, it should be underscore between words
```
private static final double RATIO_TRAIN_TEST_SPLIT = 0.8;
private static final int EPOCH = 1000;
```

go read about it , there a fixed convention on why this is shaped this way on the internet

**Review by codenamewei — CHANGES_REQUESTED** [review]

**Comment by KianYang-Lee** [comment]
Updated the binary classification solution. Also included exercise code for the solution to be attempted by students. Kindly review. Thank you.

**Comment by codenamewei** [comment]
@KianYang-Lee one more thing, remember to update readme referring to this program
https://github.com/CertifaiAI/TrainingLabs/blob/master/README.md

**Comment by codenamewei** [comment]
@KianYang-Lee need to click this or tag the individual in comments to get notified 
Capture

**Review by codenamewei — CHANGES_REQUESTED** [review]

File: dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java (hunk 1 above)
Comment (KianYang-Lee) [reply dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java]: Done

File: dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java (hunk 2 above)
Comment (KianYang-Lee) [reply dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java]: Done @codenamewei

File: dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java (hunk 3 above)
Comment (KianYang-Lee) [reply dl4j-labs/src/main/java/ai/certifai/training/classification/heartDiseaseBinaryClassification.java]: Done @codenamewei

**Comment by KianYang-Lee** [comment]
Updated header file, modified certain cases, and added file description as per suggested, please review @codenamewei

**Pull request closed without merging**